vibe_code_apps_22/scripts/freeze_eplus_baseline.py

```
@@ -0,0 +1,201 @@
+            prev_fp = prev.get("baseline_fingerprint_sha256")
+            if prev_fp and prev_fp != report["baseline_fingerprint_sha256"]:
+                archive = out_site / f"immutable_baseline_superseded_{prev_fp[:16]}.json"
+                if not archive.is_file():
+                    archive.write_text(path.read_text(encoding="utf-8"), encoding="utf-8")
+                report["supersedes_fingerprint"] = prev_fp
+                report["immutability_note"] = (
+                    "Prior baseline archived; this file is a new freeze pointer, "
+                    "not an in-place silent rewrite of the same scientific record."
+                )
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)